faker/providers/person/en_NG/__init__.py

```diff
@@ -0,0 +1,241 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+from .. import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    formats = (
+        '{{first_name_male}} {{last_name}}',
+        '{{first_name_male}} {{last_name}}',
+        '{{first_name_male}} {{last_name}}',
+        '{{first_name_male}} {{last_name}}',
+        '{{first_name_male}} {{last_name}}-{{last_name}}',
+        '{{first_name_female}} {{last_name}}',
+        '{{first_name_female}} {{last_name}}',
+        '{{first_name_female}} {{last_name}}',
+        '{{first_name_female}} {{last_name}}',
+        '{{first_name_female}} {{last_name}}-{{last_name}}',
+        '{{prefix_male}} {{first_name_male}} {{last_name}}',
+        '{{prefix_female}} {{first_name_female}} {{last_name}}',
+        '{{prefix_male}} {{first_name_male}} {{last_name}}',
+        '{{prefix_female}} {{first_name_female}} {{last_name}}',
+    )
+
+    # Names from
+    # https://andela-celisha-wigwe.github.io/names.html
```

Comment:
This link results in a 404 for me. We can't use sources that are not public.

Reply:
The user on GitHub changed their username and since GitHub doesn't reserve the old name there's no automatic redirect. The user formerly known as `andela-celisha-wigwe` is now known as `elchroy`:

```suggestion
    # https://elchroy.github.io/names.html
```

Reply:
This appear to be just a list of names that someone compiled without any mention of its statistical basis.

Are there any more "official" sources available? A governmental office of statistics, or a research paper would work great.

Reply:
There's really no official list online, not even on Wikipedia. I've added 2 sources though. But for some weird reason, this pull request is not been updated with my new commits. Had to create another PR https://github.com/joke2k/faker/pull/1042